packages/vm/Cargo.toml

```diff
@@ -7,6 +7,15 @@ description = "VM bindings to run cosmwams contracts"
 repository = "https://github.com/CosmWasm/cosmwasm/tree/main/packages/vm"
 license = "Apache-2.0"
 
+[package.metadata.release]
+pre-release-replacements = [
+    { file="../../CHANGELOG.md", search="Unreleased", replace="{{version}}" },
+    { file="../../CHANGELOG.md", search="\\.\\.\\.HEAD", replace="...{{tag_name}}", exactly=1 },
+    { file="../../CHANGELOG.md", search="ReleaseDate", replace="{{date}}" },
```

Comment:
If we do want the replacements, could we just replace the `## [Unreleased]` with the full `## [{{version}}] - {{date}}`? Having the `[Unreleased] - ReleaseDate` in the changelog reads a little weird

Reply:
Yep, we could. Implemented that